CHANGELOG.md

```diff
@@ -1,10 +1,11 @@
 # Changelog
 
-## [10.6.0] - 2020-02-04
+## [10.7.0] - 2020-02-04
```

Comment:
Should still be 10.6 if it wasn't published to npm?

Reply:
good call